### ISSUE TITLE
Setting microbit_runtime_hex in settings.json has no effect

### ISSUE THREAD

**Issue by WayneKeenan** [mu-editor/mu#435]
I'm trying to get `mu` to use a custom firmware by setting `microbit_runtime_hex ` in my`~/Library/Application\ Support/mu/settings.json` file, but it has no effect, examples:

```
{
  "theme": "day",
  "workspace": "/Users/wayne/mu_code",
  "paths": [],
  "microbit_runtime_hex": "microbit-blue-pairing-not-required.hex"
}
```

and

```
{
  "theme": "day",
  "workspace": "/Users/wayne/mu_code",
  "paths": [],
  "microbit_runtime_hex": "/Users/wayne/Library/Application Support/mu/microbit-blue-pairing-not-required.hex"
}
```

**Comment by ntoll** [comment]
This is duplicate of #434

**Comment by ntoll** [comment]
Sorry... my mistake. This needs investigating.

**Comment by ntoll** [comment]
This has been fixed by  bf8bb75 which just landed in master. It'll be in beta.16
